host-controller/src/main/java/org/jboss/as/host/controller/logging/HostControllerLogger.java

```diff
@@ -1400,4 +1400,12 @@ void noDomainControllerConfigurationProvidedForAdminOnly(String policyAttribute,
     @Message(id = 205, value = "An error occurred setting the -Dlogging.configuration property for server %s. Configuration path %s")
     void failedToSetLoggingConfiguration(@Cause Throwable cause, String serverName, File path);
 
+    @Message(id = 206, value = "File %s already exists, you must use --remove-existing-domain-config to overwrite existing files.")
+    IllegalStateException cannotOverwriteDomainXmlWithEmpty(String filename);
+
+    @Message(id = 207, value = "File %s already exists, you must use --remove-existing-host-config to overwrite existing files.")
+    IllegalStateException cannotOverwritHostXmlWithEmpty(String filename);
```

Comment:
1) There's a missing 'e' in Overwrit

2) This method is not used. There's somewhere else where I was doubting the logger call it made; probably it should have called this one and not cannotOverwriteDomainXmlWithEmpty.